CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
 - [FIXED] Threading issues in replicators.
 - [REMOVED] Removed deprecated class `CDTSavedHTTPAttachment` and method
   `createRevisionFromJson` on `CDTDocumentRevision`.
+- [FIXED] Added status code `TDReplicatorErrorNetworkOffline` -
+  replicators will go into an error state with this error code if the
+  network goes offline
```

Comment:
This probably needs an "instead of appearing to complete normally" so that people understand the change of the fix.
And a full stop.